src/__tests__/testScreen.py

```diff
@@ -77,16 +77,16 @@ def compareToExpected(self, testName, actualLines):
         self.assertEqual(
             len(actualLines),
             len(expectedLines),
-            'Actual lines was %d but expected lines aws %d' % (
-                len(actualLines), len(expectedLines)),
+            'In file %s, actual lines was %d but expected lines was %d' % (
+                testName, len(actualLines), len(expectedLines)),
```

Comment:
technically this is the test name and not the filepath, not sure if we want to make that clear